client/lib/feature-detection/index.js

```diff
@@ -1,19 +1,15 @@
 /** @format */
 
-const featureDetection = {
-	/**
-	 * Detects if CSS custom properties are supported
-	 *
-	 * @return {boolean} true when feature is supported
-	 */
-	supportsCssCustomProperties: () => {
-		return (
-			typeof window !== 'undefined' &&
-			window.CSS &&
-			window.CSS.supports &&
-			( window.CSS.supports( '--a', 0 ) || window.CSS.supports( 'color', 'var(--a)' ) )
-		);
-	},
-};
-
-export default featureDetection;
+/**
+ * Detects if CSS custom properties are supported
+ *
+ * @return {boolean} true when feature is supported
+ */
+export function supportsCssCustomPropertie() {
```

Comment:
Wee nit: This should be named `supportsCssCustomProperty` no?

Reply:
not sure how that slipped by - good catch!

Reply:
existing was `properties`

Reply:
Ahh yes that's right :) Thanks for the quick edit!